components/user/UserTabs.js

```diff
@@ -92,7 +92,7 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
           name="tabs"
           onChange={changeTab}
           className="block w-full rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
-          defaultValue={tabs.find((tab) => tab.current).name}
+          value={tabs.find((tab) => tab.current).name}
```

Comment:
This change should not be part of this pull request

Reply:
actually I resolved two issues in one pr 
because the changes were too little

Reply:
little changes are good, otherwise it is more difficult for maintainers